include/deal.II/numerics/vector_tools_boundary.h

```diff
@@ -687,6 +687,9 @@ namespace VectorTools
    * hp-namespace version of project_boundary_values_curl_conforming_l2
    * (above).
    *
+   * This function has not been instantiated, you may need the templates header
+   * to use it.
```

Comment:
I don't think that it is worthy to instantiate this function, not many people are going to use it. I would just add a comment. What do you think?

Reply:
We can get rid of this now that #10667 is in.